doc/classes/ProjectSettings.xml

```diff
@@ -1628,8 +1628,11 @@
 		</member>
 		<member name="rendering/2d/snap/snap_2d_vertices_to_pixel" type="bool" setter="" getter="" default="false">
 		</member>
-		<member name="rendering/anti_aliasing/quality/msaa" type="int" setter="" getter="" default="0">
-			Sets the number of MSAA samples to use (as a power of two). MSAA is used to reduce aliasing around the edges of polygons. A higher MSAA value results in smoother edges but can be significantly slower on some hardware. See also bilinear scaling 3d [member rendering/scaling_3d/mode] for supersampling, which provides higher quality but is much more expensive.
+		<member name="rendering/anti_aliasing/quality/msaa_2d" type="int" setter="" getter="" default="0">
+			Sets the number of MSAA samples to use for 2D/Canvas rendering (as a power of two). MSAA is used to reduce aliasing around the edges of polygons. A higher MSAA value results in smoother edges but can be significantly slower on some hardware.
```

Comment:
We should put together a list of what objects users can expect to see AA on and which they won't. For example, users shouldn't expect AA either within or on the edges of sprites. But they may see AA on the edges of sprites where the Texture us solid and goes right to the edge of the quad.

Reply:
Shouldn't MSAA work on the edges of sprites (at least if they're rotated)? Sprites are rects after all, and they're probably drawn with 2 triangles behind the scenes.

Reply:
Added a small note to all MSAA related properties: `This has no effect on shader-induced aliasing or texture aliasing.`